beacon-chain/blockchain/chain_info.go

```diff
@@ -380,6 +381,14 @@ func (s *Service) InForkchoice(root [32]byte) bool {
 	return s.cfg.ForkChoiceStore.HasNode(root)
 }
 
+// IsViableForkCheckpoint returns whether the given checkpoint is a checkpoint in any
+// chain known to forkchoice
+func (s *Service) IsViableForCheckpoint(cp *forkchoicetypes.Checkpoint) (bool, error) {
```

Comment:
I am not seeing that this function is called from anywhere

Reply:
+1. I think it can be removed